src/AndroidClient/TacControl/TacControl/Views/MapPage.xaml.cs

```diff
@@ -136,7 +133,7 @@ private void GenerateLayers(List<Helper.SvgLayer> layers)
                 layerLoadTasks.Add(
                     Task.Run(() =>
                     {
-                        using (var stream = new MemoryStream(Encoding.UTF8.GetBytes(svgLayer.content)))
+                        using (var stream = svgLayer.content.GetStream())
```

Comment:
This has probably better performance right? should probably do same in desktpop client.

Reply:
I think it's the same in the desktop client today